src/runtime/less/ts-forms.less

```diff
@@ -121,12 +121,13 @@
 			font-size: @ts-fontsize-mini;
 			font-weight: @ts-fontweight;
 			border-radius: @ts-radius;
-			padding-top: @ts-unit-quarter - 1px;
-			padding-bottom: @ts-unit-quarter - 1px;
-			padding-left: @ts-unit-half;
-			padding-right: 0;
+			padding: @ts-unit-quarter - 1px @ts-unit-half;
```

Comment:
I think this one is not clear what it does. does this count as 3 argument or 2? 
I would stick to the separate property for this one, like before.

Reply:
👍